server/controllers/notification.js

```diff
@@ -0,0 +1,62 @@
+const asyncHandler = require("express-async-handler");
+const Notification = require("../models/Notification");
+
+//@desc Create notification
+//@route POST /notifications
+//@ access Private
+exports.createNotification = asyncHandler(async (req, res, next) => {
+  const { title, description, read } = req.body;
+
+  const notification = await new Notification({
+    title,
+    description,
+    read,
+    userId: req.user.id,
```

Comment:
Won't the `userId` here be the recipient's id and not that of the logged in user? :)

Reply:
@rajivtitus , I think this method is creating notification by logged in user, I seem not to understand where the recipient is coming from. Kindly explain better please.

Reply:
@sholabioku You can send the `userId` in the request body as opposed to using the logged in user's id.

Take for example, a message being sent from user A to B. We would need to notify user B that user A has sent them a message. However, when we create a notification when sending a message, the notification will be created for the logged in user (user A) - which is not intended